src/navigation/optOutFooter.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+// this provides an opt-out link in the footer for the compulsory opt-in phase of Next
+module.exports = function (items, currentUrl) {
+	const optOutLink = {
+		name: 'Old FT.com',
+		href: `/opt-out-confirm?location=${encodeURIComponent(currentUrl)}`
+	};
+	const supportBlock = items.find(i => {
+		return i.title === 'Support';
+	});
+	if (supportBlock) {
+		supportBlock.optOutLink = optOutLink;
```

Comment:
ah - this module receives `items` as an argument (items are 'support', 'legal and services', 'tools' etc), looks for the one title support, then adds the optout link to that block, before returning the amended `items`. Footer has items which have their own items.